### ISSUE TITLE
Interesting use case (sharding, Oracle import)

### ISSUE THREAD

**Issue by sorin-pop** [djavorszky/ddn#281]
We received an Oracle dump from a a customer which uses two shards (so two different Oracle schemas). In the dump they exported both schemas, something like this:

ystem/********@intra1 directory=dumpdir dumpfile=save_liferay_test.dmp logfile=save_liferay_test.log schemas=lpextra_test1,lportal_test1

The CloudDB oracle import **is** prepared for the case when the dump contains more than one schema, and what it actually does is that it identifies all the included schemas and maps them all to the single schema that we import into.

However, this time this proved problematic, because these two schemas now, being two shards actually, contain the same Liferay tables. So basically what happened at import time, as far as I could find out, is that the rows from one schema were imported, and then when it was the turn of the second one, its rows were skipped, because the tables already existed. Because I see lines like these in the import log:

ORA-39151: Table "MERKURINTRA11SEBI"."ACCOUNT_" exists. All dependent metadata and data will be skipped due to table_exists_action of skip
ORA-39151: Table "MERKURINTRA11SEBI"."ADDRESS" exists. All dependent metadata and data will be skipped due to table_exists_action of skip

See attached the whole import log. 

[MERKURINTRA11SEBI.LOG](https://github.com/djavorszky/ddn/files/2016502/MERKURINTRA11SEBI.LOG)

In a nutshell, the rows of only one shard were imported, I think. That's why the CSE who did the import could not find certain rows/data that he was looking for.

For now I just wrote this down for the record. Later we can think about it. (perhaps we should run the import with that table_exists_action set in such a way that the rows are not skipped if the table already exists...)

**Comment by sorin-pop** [comment]
Due to lack of feedback, I am closing this ticket (and make no changes).
